.circleci/config.yml

```
@@ -33,30 +33,61 @@ commands:
             set -euo pipefail
-            cleanup() { docker rm -f << parameters.name >> >/dev/null 2>&1 || true; }
+            cleanup() { docker rm -f << parameters.container_name >> >/dev/null 2>&1 || true; }
             trap cleanup EXIT
-            docker run -d --rm -p << parameters.host_port >>:<< parameters.port >> --name << parameters.name >> << parameters.image >>
+            docker run -d --rm -p << parameters.host_port >>:<< parameters.port >> --name << parameters.container_name >> << parameters.image >>
+
+            # Debug: Check container started successfully
+            echo "🔍 Debug: Container started, checking status..."
+            docker ps -a | grep << parameters.container_name >> || echo "Container not found in docker ps"
+            echo "🔍 Debug: Port mapping should be << parameters.host_port >>:<< parameters.port >>"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)